backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4023,4 +4023,86 @@ databaseChangeLog:
               LEFT JOIN ${database.defaultSchemaName}.result ON t.internal_id=${database.defaultSchemaName}.result.test_event_id 
               WHERE ${database.defaultSchemaName}.result.test_event_id IS NULL AND t.result IS NOT NULL;
       rollback:
-        sql: SELECT 'foo';
\ No newline at end of file
+        sql: SELECT 'foo';
+  - changeSet:
+      id: create-feature-flag-table
```

Comment:
Can you update the id so it's the same as the tag? It makes rollbacks a little easier